docs/daemon/cli.mdx

```diff
@@ -0,0 +1,84 @@
+---
+sidebar_position: 2
+sidebar_label: "CLI"
+id: daemon-cmdline
+title: Kwild CLI Guide
+description: Kwild Command Line
+slug: /daemon/cmdline
+---
+
+Kwild command starts both the blockchain and the database services of kwild node. 
+
+### Quickstart a kwild node:
+Run the below command to quickstart a kwild node without specifying any configuration.
+```bash
+kwild --autogen  --root_dir "path/to/root/dir"
+```
+This command will generate the missing configuration in the root directory and starts the node with the generated configuration. 
+
+The default root directory used is `~/.kwild` and can be overwritten using `--root_dir`
+
+### Run node with specific configuration
+If root directory is already intialized with the required configuration, then node can be started just by running the below command:
```

Comment:
I think we should strive for proper grammar throughout the docs.  Things like "If the root directory... then the node can be started by running the below command."